package.json

```diff
@@ -49,10 +49,11 @@
     "eslint-plugin-node": "^6.0.1",
     "glob": "^7.1.1",
     "mocha": "^4.0.1",
-    "mocha-eslint": "^4.1.0"
+    "mocha-eslint": "^4.1.0",
+    "qunit-dom": "^0.6.2"
   },
   "engines": {
-    "node": ">= 4"
+    "node": "6.* || 8.* || >= 10.*"
```

Comment:
Oops, it looks like causes some problems. Do we want/need to keep support for node 4 to avoid a breaking change?